with-libsql/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "scripts": {
+    "start": "expo start",
+    "android": "expo run:android",
+    "ios": "expo run:ios",
+    "web": "expo start --web"
+  },
```

Comment:
```suggestion
```

Reply:
thanks Evan! i would keep android and ios still. because this will never be supported in expo-go. good to run from CNG still.